packages/python-packages/api-stub-generator/apistub/nodes/_class_node.py

```diff
@@ -220,20 +221,20 @@ def generate_tokens(self, apiview):
 
     def _generate_child_tokens(self, apiview):
         # Add members and methods
-        apiview.add_new_line()
         apiview.begin_group()
         for e in [p for p in self.child_nodes if not isinstance(p, FunctionNode)]:
+            apiview.add_newline()
             apiview.add_whitespace()
             e.generate_tokens(apiview)
-            apiview.add_new_line()
-        apiview.add_new_line(1)
+            apiview.add_newline()
```

Comment:
This will cause review to have two lines between each member within a class. right?

Reply:
No. "add_newline" is basically just a carriage return. It cannot, be design, create blank lines
```python
add_newline()
add_newline()
add_newline()
```
This has the same effect as only a single newline.